platform-api/che-core-api-account/src/test/java/org/eclipse/che/api/account/AccountServiceTest.java

```diff
@@ -368,7 +352,7 @@ public void shouldBeAbleToGetAccountByName() throws Exception {
     }
 
     @DataProvider(name = "roleProvider")
-    public String[][] roleProvider() {
+    public Object[][] roleProvider() {
```

Comment:
`@DataProvider` javadoc says:  _The annotated method must return an `Object[][]` where each `Object[]` can be assigned the parameter list of the test method._